src/Viewer.tsx

```diff
@@ -539,9 +539,9 @@ function Viewer(): ReactElement {
   }, [isUserDirectlyControllingFrameInput, frameInput]);
 
   const onClickId = useCallback(
-    (info: PixelIdInfo) => {
-      if (dataset && info.globalId !== undefined) {
-        annotationState.handleAnnotationClick(dataset, info.globalId);
+    (info: PixelIdInfo | null) => {
+      if (dataset) {
+        annotationState.handleAnnotationClick(dataset, info?.globalId ?? null);
```

Comment:
Clicks on the background (not on cells) are now passed to the annotation click handler as well. We use this to clear the editing input field!